docs/spec/bento-common-props.md

```diff
@@ -0,0 +1,23 @@
+# Common Props in Preact/React Bento Components
+
+The following are common props that should be be supported in al Bento Preact/React Bento Components when possible.
+
+## `as`
+
+Allows consumer to customize the html tag that is used to implement a component.
+
+## `class`
```

Comment:
Note that Preact supports `class` and React supports `className`

Reply:
Hmm I've been seeing existing components take `class` as a prop. So doesn't that mean that those components would break for React consumers? 

Also, what would be recommendation for a preact/react-agnostic way of accept classes as a prop?

Reply:
good question, the components should all take `class` and the react build will convert that to `className` -- this is related to #35678 which @alanorozco is working on

I think it's fine to do either of:
```
## `class` (Preact only) or `className` (React only)
```
or
```
## `class` (Preact only)

## `className` (React only)
```

Reply:
went with the latter - done